teuthology/orchestra/remote.py

```diff
@@ -160,22 +160,23 @@ def shortname(self):
 
     @property
     def is_online(self):
-        if self.ssh is None:
-            return False
-        if self.ssh.get_transport() is None:
-            return False
-        try:
-            self._runner(args="true", client=self.ssh, name=self.shortname)
-        except Exception:
+        """
+        This function does not give a comprehensive answer
+        if the connection is still alive, for this purpose
+        the one must use `ensure_online` method.
+        """
+        return self.is_active()
+
+    def is_active(self):
```

Comment:
why do we need two `is_online` ?

Reply:
BTW, i think is_online is a better name than is_active. personally, i like `is_reachable` even better.

Reply:
yeah... it is embarrassing me two... just need to be sure which signature to leave...

Reply:
I would get rid of `is_online` because looks like it is not possible to make sure it is online, so more correctly to leave `is_active`?

Reply:
an online remote machine sounds more correct than an active remote box . and again, online is objective. but reachable is more subjective. i'd say we cannot tell if a remote is online/active unless we can be sure about the networking and stuff work. and as you put, 

>  looks like it is not possible to make sure it is online